src/plugins/telemetry/server/telemetry_collection/get_local_license.ts

```diff
@@ -22,6 +22,14 @@ import { ElasticsearchClient } from 'src/core/server';
 
 let cachedLicense: ESLicense | undefined;
 
+async function fetchLicense(esClient: ElasticsearchClient, local: boolean) {
```

Comment:
@afharo here's the change requested in https://github.com/elastic/kibana/pull/78046#discussion_r493658248

Reply:
Thank you! All looking good! Let's find out what's wrong with the new failing test and we should be ready to go!

Reply:
[I couldn't resist](https://github.com/elastic/kibana/pull/78046/commits/a572d75eeafabd8008c75c5e995b6f13cde77265)